### PR TITLE
[fix](Nereids) npe when one row relation contain aggregate function

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
# Proposed changes

```sql
mysql [test]>select sum(1);
ERROR 1105 (HY000): errCode = 2, detailMessage = Unexpected exception: null
```

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by morrySnow — DISMISSED** [review]

File: regression-test/suites/tpch_sf0.1_p1/function/test_not_supported_agg_fn.groovy [hunk 1]
```diff
@@ -37,6 +37,15 @@ suite("test_not_supported_agg_fn") {
             """
             exception "errCode = 2, detailMessage = No matching function with signature: last_value(bitmap)"            
         }
+
+        sql "set enable_nereids_planner=true;"
```
Comment (morrySnow) [inline regression-test/suites/tpch_sf0.1_p1/function/test_not_supported_agg_fn.groovy:41]: it is better to move this case to `nereids_syntax_p0`

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run p1

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**